code/modules/client/preferences.dm

```diff
@@ -186,26 +186,29 @@ var/const/MAX_SAVE_SLOTS = 8
 		var/theckey = C.ckey
 		var/thekey = C.key
 		spawn()
-			while(!speciesinit)
-				sleep(1)
 			if(!IsGuestKey(thekey))
 				var/load_pref = load_preferences_sqlite(theckey)
 				if(load_pref)
-					if(!C)
-						WARNING("[__LINE__]: client C is null for [theckey].")
-					if(load_save_sqlite(theckey, C, default_slot) && C)
-						saveloaded = 1
-						return
-					else
-						world.log << "[theckey] failed loading save slot."
-				else
-					world.log << "[theckey] failed loading preferences."
+					while(!speciesinit)
```

Comment:
oh I forgot to tab it.
the sleep is supposed to be in the while but the try isn't.